modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/tasks/CreateTokenTask.java

```diff
@@ -43,7 +43,9 @@
 /**
  * @author Andrea Di Giorgi
  * @author Gregory Amerson
+ * @deprecated The token is no longer being used
  */
+@Deprecated
 public class CreateTokenTask extends DefaultTask {
```

Comment:
We should print out a warning in the createToken task in the execute just do a NOP, don't actually create the token since we do nothing with the tokens anymore.